emloop/main_loop.py

```diff
@@ -35,6 +35,7 @@ def __init__(self,   # pylint: disable=too-many-arguments
                  train_stream_name: str=EL_DEFAULT_TRAIN_STREAM,
                  extra_streams: Iterable[str]=(),  # pylint: disable=invalid-sequence-index
                  buffer: int=0,
+                 epochs_count: int=1,
```

Comment:
```suggestion
                 epochs_count: Optional[int]=None,
```
lets make this `None` by default and do not stop if its not specified